client/state/data-layer/wpcom-http/actions.js

```diff
@@ -47,7 +47,7 @@ export const http = ( {
 		body,
 		method,
 		path,
-		query: method === 'GET' ? { ...query, ...version } : version,
+		query: method === 'GET' && { ...query, version },
```

Comment:
note that we probably don't want to set `query = false` here which is what will happen if `method !== 'GET'`